client/src/components/Tickets/TicketCreateForm.jsx

```diff
@@ -197,7 +197,7 @@ function TicketCreateForm({
                       <FormField
                         name="cc"
                         label="Cc"
-                        description="Separate emails with commas"
+                        description="Separate emails with commas. NOTE: Emails listed here will only receive emails on replies to tickets."
```

Comment:
I believe we want to preserve the change to `Requestor` for the createTicket calls on the backend. It was just the other calls we did not need to change

Reply:
All fixed!